bin/cloudgov/setup-egress-for-apps

```diff
@@ -141,10 +141,7 @@ $output cf target -o "$startorg" -s "$startspace" 2>&1 > /dev/null
 for app in $applist; do
     echo "Configure Egress settings for $app"
     $output cf add-network-policy "$app" "$proxyname" -s "$egressspace" -o "$egressorg" --protocol tcp --port 61443
-    $output cf unset-env "$app" http_proxy  # $proxyroute
```

Comment:
Just as a heads-up, we added these lines to explicitly clear these http[s]_proxy environment variables when we realized they were messing up Drupal's ability to retrieve s3 files. 

It's fine to delete these lines as long as we don't have something else setting them again. If we've missed something like that somewhere, problems with newly-uploaded images will be the symptom.